Hudl.Mjolnir/Command/Command.cs

```diff
@@ -169,7 +177,8 @@ internal Command(string group, string name, string breakerKey, string poolKey, T
                 throw new ArgumentNullException("poolKey");
             }
 
-            if (defaultTimeout.TotalMilliseconds <= 0)
+            TimeoutsIgnored = IgnoreCommandTimeouts.Value;
+            if (!TimeoutsIgnored && defaultTimeout.TotalMilliseconds <= 0)
```

Comment:
That's true! I think there's actually a test which looks at this. It will give a false positive in the case where ignoredTimeouts is true and the timeout is <=0